app/views/projects/_form.html.erb

```diff
@@ -14,10 +14,19 @@
   <div class="form-group">
     <%= form.label :name, "Project name:", class: "control-label" %>
     <%= form.text_field :name, class: "form-control" %>
+    <%= form.label :description, "Project description:", class: "control-label" %>
+    <%= form.text_field :description, class: "form-control" %>
```

Comment:
You can also use `form.text_area` for fields with longer test (it will look nicer and you can resize it)